front/src/app/feature/admin-management/components/team/members-card/components/delete-popup/delete-popup.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, EventEmitter, Input, Output, SimpleChanges} from '@angular/core';
+import {TeamMember} from '../../../../../type/team/team-member';
+
+@Component({
+  selector: 'app-delete-popup',
+  standalone: true,
+  imports: [],
+  templateUrl: './delete-popup.component.html',
+  styleUrl: './delete-popup.component.scss'
+})
+export class DeletePopupComponent {
+  @Input() member: TeamMember | null = null;
+  @Input() isDeleting : boolean = false;
+  @Input() isOpen : boolean = false;
+
+  @Output() confirm = new EventEmitter<TeamMember>();
+  @Output() cancel = new EventEmitter<void>();
+
+  memberName: string = '';
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes['member'] && this.member) {
+      this.memberName = this.member.displayName || this.member.email || 'this member';
+    }
+  }
```

Comment:
todo: une fois que member sera passé en input()

```suggestion
  memberName = computed(() => this.member().displayName || this.member().email || 'this member')
```